Makefile

```diff
@@ -7,6 +7,8 @@ IMAGE_TAG          ?= latest
 OPERATOR_IMAGE     ?= kubevirt/hyperconverged-cluster-operator
 REGISTRY_NAMESPACE ?=
 
+SHA:=$(shell git describe --no-match  --always --abbrev=40 --dirty)
```

Comment:
nit: you should put this up with, and aligned, with the rest.

Reply:
thanks will do